source/reference/configure-shell-settings-api.txt

```diff
@@ -0,0 +1,280 @@
+.. _configure-settings-api:
+
+==============================================
+Configure :binary:`~bin.mongosh` Using the API
```

Comment:
[Note for review]

The content on this page was taken from this page: https://docs.mongodb.com/mongodb-shell/reference/configure-shell-settings/

The only differences are:

- I've moved the "Customize the Prompt" section on the page to its own sub-page.
- I've added a new section comparing the behavior of the config API and config file. See [Behavior with Configuration File](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-20841/reference/configure-shell-settings-api/#behavior-with-configuration-file)